app/src/components/user/AccountIndexPage/ManageAccount/ManageTeams/SwitchWorkspaceModal/SwitchWorkspaceModal.tsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import { Avatar, Button, Col, Row } from "antd";
+import { PlusOutlined } from "@ant-design/icons";
+import { RQModal } from "lib/design-system/components";
+import LearnMoreAboutWorkspace from "../TeamViewer/common/LearnMoreAboutWorkspace";
+import { getUniqueColorForWorkspace } from "utils/teams";
+import { Team } from "types";
+import { useSelector } from "react-redux";
+import { getAvailableTeams } from "store/features/teams/selectors";
+import { actions } from "store";
+import { useDispatch } from "react-redux";
+import { switchWorkspace } from "actions/TeamWorkspaceActions";
+import { getAppMode, getUserAuthDetails } from "store/selectors";
+import "./switchWorkspaceModal.css";
+
+interface SwitchWorkspaceModalProps {
+  isOpen: boolean;
+  toggleModal: () => void;
+}
+
+const SwitchWorkspaceModal: React.FC<SwitchWorkspaceModalProps> = ({ isOpen, toggleModal }) => {
+  const dispatch = useDispatch();
+
+  const availableTeams = useSelector(getAvailableTeams);
+  const appMode = useSelector(getAppMode);
+  const user = useSelector(getUserAuthDetails);
+
+  const handleCreateNewWorkspaceClick = () => {
+    toggleModal();
+    dispatch(
+      actions.toggleActiveModal({
+        modalName: "createWorkspaceModal",
+        newValue: true,
+      })
+    );
+  };
+
+  const handleSwitchWorkspaceClick = (team: Team) => {
+    toggleModal();
+    switchWorkspace(
+      {
+        teamId: team.id,
+        teamName: team.name,
+        teamMembersCount: team.accessCount,
+      },
+      dispatch,
+      {
+        isSyncEnabled: user?.details?.isSyncEnabled,
+        isWorkspaceMode: true,
+      },
+      appMode
+    );
+    dispatch(
+      actions.toggleActiveModal({
+        modalName: "inviteMembersModal",
+        newValue: true,
+      })
+    );
```

Comment:
let's not show the invites modal when the user is switching to a workspace where he/she is not an admin

https://github.com/requestly/requestly/assets/61556757/7f524e5e-076b-47d2-9833-5809c5ffb9aa

Reply:
Can we also reduce some space below the list maybe `1rem`
<img width="760" alt="Screenshot 2023-07-07 at 4 02 47 PM" src="https://github.com/requestly/requestly/assets/61556757/c0be13d0-275c-4741-a60a-eddcd893c2f3">

Reply:
Showing that message seemed best in this case because the CTA was invite so expectation is to invite

Reply:
then I think we need to update the copy for switch workspace modal eg `You need to be in a workspace to invite others` something like that wdyt ?